yarn.lock

```diff
@@ -10677,15 +10677,15 @@ prettier-linter-helpers@^1.0.0:
   dependencies:
     fast-diff "^1.1.2"
 
-prettier@^1.14.3:
-  version "1.15.3"
-  resolved "https://registry.yarnpkg.com/prettier/-/prettier-1.15.3.tgz#1feaac5bdd181237b54dbe65d874e02a1472786a"
-
-prettier@^2.0.5:
+prettier@2.0.5:
   version "2.0.5"
   resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.0.5.tgz#d6d56282455243f2f92cc1716692c08aa31522d4"
   integrity sha512-7PtVymN48hGcO4fGjybyBSIWDsLU4H4XlvOHfq91pz9kkGlonzwTfYkaIEwiRg/dAJF9YlbsduBAgtYLi+8cFg==
 
+prettier@^1.14.3:
```

Comment:
I don't understand why there's two entries for prettier with different versions?

Reply:
This has almost certainly to do with that some other package requires prettier of a certain version. Node is good about that kind of version management. I generally never review the yarn.lock file because it has a ton of information like this which is hard to read and is automatically generated when we run yarn install. The file package.json is the one it bases all of its resolutions on.

Reply:
OK!